assets/js/modules/adsense/datastore/accounts.js

```diff
@@ -88,13 +99,15 @@ export const actions = {
 
 export const controls = {
 	[ FETCH_ACCOUNTS ]: () => {
-		return API.get( 'modules', 'adsense', 'accounts' );
+		return API.get( 'modules', 'adsense', 'accounts', undefined, {
```

Comment:
Will this take `null` as an argument for `queryParams` instead of `undefined`? I know it's functionally the same, but it might make the code a bit clearer to read is all.

If not, no worries for now.

Reply:
I chose to use `undefined` just to make sure it's exactly the same value as there would be otherwise. Probably `null` would work, but I'd say it's not a big deal anyway :)